scripts/template_changelog.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "<% X.Y.Z %>"
+description: "Released on <% MM/DD/YYYY %>"
+---
+
+### Breaking Changes ❗
```

Comment:
I love the cutesy emojis btw

@khorne3 mentioned we should use sentence case for lower-level headings, so I think this would be "Breaking changes" ?

Reply:
If we do that, we should update all clogs. Happy for us to do that, though as a different changeset. I'm super glad we're increasing our consistency and standardizing these decisions!!!!! :heart:

Reply:
btw, and it pains me a bit to say this, because I love the emojis, but... the emojis result in slightly worse fragments: `https://coder.com/docs/changelog/1.17.0#breaking-changes-` (note the trailing hyphen)

I dunno if there's some way to customize this in the JavaScript we're using to transform the Markdown, though

Reply:
Worth investigating - CH story would be useful as that's the responsibility of `m`, not anything in `cdr/docs` AFAICT